packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpJesApi.ts

```diff
@@ -20,143 +20,165 @@ import { AbstractFtpApi } from "./ZoweExplorerAbstractFtpApi";
 // The Zowe FTP CLI plugin is written and uses mostly JavaScript, so relax the rules here.
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
-/* eslint-disable @typescript-eslint/no-unsafe-call */
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/no-unused-vars*/
 
 export class FtpJesApi extends AbstractFtpApi implements ZoweExplorerApi.IJes {
     public async getJobsByOwnerAndPrefix(owner: string, prefix: string): Promise<zowe.IJob[]> {
         const result = this.getIJobResponse();
-        const connection = await this.ftpClient(this.checkedProfile());
-        if (connection) {
-            const options = {
-                owner: owner,
-            };
-            const response = await JobUtils.listJobs(connection, prefix, options);
-            if (response) {
-                const results = response.map((job: IJob) => {
-                    return {
-                        ...result,
-                        /* it’s prepared for the potential change in zftp api, renaming jobid to jobId, jobname to jobName. */
-                        jobid: (job as any).jobId || job.jobid,
-                        jobname: (job as any).jobName || job.jobname,
-                        owner: job.owner,
-                        class: job.class,
-                        status: job.status,
-                    };
-                });
-                return results;
+        let connection: any;
+        try {
+            connection = await this.ftpClient(this.checkedProfile());
+            if (connection) {
+                const options = {
+                    owner: owner,
+                };
+                const response = (await JobUtils.listJobs(connection, prefix, options)) as IJob[];
```

Comment:
When running `yarn package`, I received the following ESLint error:

<img width="680" alt="Screen Shot 2021-06-04 at 3 56 13 PM" src="https://user-images.githubusercontent.com/45975633/120861535-745bd700-c555-11eb-86ec-b4f2bb39e632.png">


To resolve this error, I had to delete `as IJob[]` from the end of line 34.

Reply:
Yes, I think `as IJob[]` is unnecessary now. `JobUtils.listJobs` returns `Promise<IJob[]>`. Thanks!
```
public static async listJobs(connection: any, prefix: string, option?: IListJobOption): Promise<IJob[]>
```

Reply:
Thank you @std4lqi for confirming this. I've pushed a commit removing `as IJob[]` to the PR branch. (@tiantn I hope this is ok with you! I just figured it would be quicker to do a direct push since this was a small, low-risk change.)